CHANGELOG.md

```diff
@@ -1,5 +1,86 @@
 # [Versions](https://mui.com/versions/)
 
+## v5.15.5
+
+<!-- generated comparing v5.15.4..master -->
+
+_Jan 17, 2024_
+
+A big thanks to the 16 contributors who made this release possible. Here are some highlights ✨:
+
+- Base UI's CSS class prefix is now `base-` (#40205) @michaldudak
+- Bug fixes, and a lot of improvements to code and docs infra
+
+### `@mui/material@5.15.5`
```

Comment:
I don't see https://github.com/mui/material-ui/pull/40418, can you check?

Reply:
Updated!